src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -829,9 +829,6 @@ def _flexible_server_params(command_group):
                 c.argument('public_access', options_list=['--public-access'], arg_type=get_enum_type(['Enabled', 'Disabled']),
                            help='Determines the public access. ')
 
-        with self.argument_context('{} flexible-server replica stop-replication'.format(command_group)) as c:
-            c.argument('server_name', arg_type=server_name_arg_type)
```

Comment:
Why remove this parameter definition?

Reply:
Redundant, is already stated to be needed for all postgres flexible-server commands at line [547](https://github.com/Azure/azure-cli/blob/3b47eed45a24512fe5494555fe28f32f29a1e870/src/azure-cli/azure/cli/command_modules/rdbms/_params.py#L547)
<img width="1523" alt="image" src="https://github.com/Azure/azure-cli/assets/69922333/eaecfdd1-0045-459f-8096-7d368b6f0211">